### PR TITLE
feat: add multi-search dropdown in mobile

### PR DESCRIPTION
**Pull request by jeanphilippeds**
Hello,

A small PR to fix https://github.com/c2corg/c2c_ui/issues/2451

With selected tags:

With multi-search results:

I'm not a big fan of repeating the `filter` icon. Maybe we could put another one.

### CODE REVIEW

File: src/views/documents/utils/QueryItems.vue [hunk 1]
```diff
@@ -17,17 +17,12 @@
         <span slot="button" class="button is-size-7-mobile" :disabled="category.fields.length === 0">
           <fa-icon :icon="$options.categoryIcon[category.name]" />
           <span class="is-hidden-mobile">
-            <!-- $gettext('General') -->
-            <!-- $gettext('ratings') -->
-            <!-- $gettext('Terrain') -->
-            <!-- $gettext('Miscs') -->
```
Comment (brunobesson) [inline src/views/documents/utils/QueryItems.vue:23]: Is thee a specific reason for removing these comments? They are i18n-related, and should not be removed (except if unused). See https://github.com/c2corg/c2c_ui/wiki/i18n

**Review by brunobesson — APPROVED** [review]
Other than the removed i18n stuff, looks good to me.

File: src/views/documents/utils/QueryItems.vue (hunk 1 above)
Comment (jeanphilippeds) [reply src/views/documents/utils/QueryItems.vue:23]: Ok my bad, I thought it was dead code

**Comment by brunobesson** [comment]
There might be some problems when the criteria title is larger than the phone width:

image

**Comment by jeanphilippeds** [comment]
Good catch!
I've added some commits. I also fixed the margins between tags when we have multiple lines.

**Comment by brunobesson** [comment]
Nice job! I'll merge.

**Pull request merged**